packages/client/src/views/RegistrationHome/RowHistoryView.tsx

```diff
@@ -392,9 +392,14 @@ export class RowHistoryViewComponent extends React.Component<IProps> {
                         <>
                           <LabelValue
                             label={intl.formatMessage(constantsMessages.update)}
-                            value={this.props.intl.formatMessage(
-                              getRejectionReasonDisplayValue(rejectReasons)
-                            )}
+                            value={rejectReasons
```

Comment:
Can we introduce an another wrapper function on top of `getRejectionReasonDisplayValue` which will be able to handle array of string ?